docs/pages/api/getting-started.mdx

```diff
@@ -57,6 +57,19 @@ $ go mod init client-demo
 $ go get github.com/gravitational/teleport/api/client
 ```
 
+<Admonition type="tip" title="API Version">
+To ensure compatibility, you should use a version of the API that matches the major version of
+Teleport running in your cluster.
+
+To find the pseudoversion appropritate for a go.mod file for a specific git tag,
+run the following command from the `teleport` repository:
+
+```code
+$ go list -f '{{.Version}}' -m "github.com/gravitational/teleport/api@$(git rev-parse v12.1.0)"
```

Comment:
Should we replace `12.1.0` with a variable so that the example doesn't start to look stale in the future?

```suggestion
$ go list -f '{{.Version}}' -m "github.com/gravitational/teleport/api@$(git rev-parse v12.1.0)"
```

Reply:
We can, though it might actually be a good thing that it references an older version that is not the latest, as that's the whole point of this tip.

No strong feelings either way, WDYT?

Reply:
Fair point. As you were, sir.